### PR TITLE
Removing blueprint and moving dependencies to npm

### PR DESCRIPTION
**Pull request by gmurphey**
With ember-cli moving away from bower dependencies, thought it would be beneficial to move ember-hifi's dependencies over to npm.

### CODE REVIEW

**Comment by noslouch** [comment]
This looks great, but I'd like for it to run on circle before we merged. I just updated our setting so that forks can run in our build system; would you mind adding a new commit and pushing again in order to kick off a build?

I think an update to the readme to take out the generator command should do the trick ;)

**Comment by gmurphey** [comment]
I just realized that `app.import` can't use `node_modules` path. I'll update this to build a vendor tree.

**Comment by gmurphey** [comment]
I'm not sure what the `node-sass` errors are about, but I see them on a clean master run of `yarn test` as well. Will try to dig into this later.

**Comment by gmurphey** [comment]
So I think I've narrowed these failures down to an issue with yarn removing node-sass artifacts after some of the ember-try scenarios (https://github.com/yarnpkg/yarn/issues/3506). I was able to reproduce locally, and it seems, from my limited testing, that this is fixed in yarn 0.27 (pre-release, atm). I'm not sure if it's possible to have circleci use a specific version of yarn. If not, I'd be happy to submit a PR that has circleci use npm (which doesn't face this issue) until circleci is updated.

**Comment by noslouch** [comment]
Oh boy. Thanks for tracking this down. For every problem yarn solves, it seems to introduce some new ones. Such is the way with technology tho. I'll see if I can update which yarn circle uses.

excuse the brevity -- sent via mobile

On Jun 26, 2017, at 6:41 PM, Garrett Murphey <notifications@github.com<mailto:notifications@github.com>> wrote:


So I think I've narrowed these failures down to an issue with yarn removing node-sass artifacts after some of the ember-try scenarios (yarnpkg/yarn#3506<https://github.com/yarnpkg/yarn/issues/3506>). I was able to reproduce locally, and it seems, from my limited testing, that this is fixed in yarn 0.27 (pre-release, atm). I'm not sure if it's possible to have circleci use a specific version of yarn. If not, I'd be happy to submit a PR that has circleci use npm (which doesn't face this issue) until circleci is updated.

File: package.json [hunk 1]
```diff
@@ -59,9 +59,13 @@
     "web"
   ],
   "dependencies": {
+    "broccoli-funnel": "^1.2.0",
+    "broccoli-merge-trees": "^1.2.0",
```
Comment (noslouch) [inline package.json]: should these go into `devDependencies`?

**Comment by noslouch** [comment]
I added a command to our circle config that uses that yarn pre release and everything's looking green!

One thing: I'm not :100: on how `dependencies` vs `devDependencies` affects upstream apps, so I'm curious if those broccoli deps would affect app builds.

let me know your thoughts on that question and I'll merge 'n' release.

File: package.json (hunk 1 above)
Comment (gmurphey) [inline package.json]: The consuming apps will need these to build the addon's vendor tree. If they're in `devDependencies`, they won't be available to the app at build-time.

**Pull request merged**